### PR TITLE
update version alpha -> beta

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/AEPSampleApp/package.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
   },
   "dependencies": {
-    "@adobe/react-native-aepcore": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepuserprofile": "^1.0.0-alpha.3",
-    "@adobe/react-native-aepedge": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepedgeidentity": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepedgeconsent": "^1.0.0-alpha.1",
-    "@adobe/react-native-aepmessaging": "^1.0.0-alpha.2",
-    "@adobe/react-native-aepassurance": "^3.0.0-alpha.0",
+    "@adobe/react-native-aepassurance": "^3.0.0-beta.1",
```
Comment (emdobrin) [inline apps/AEPSampleApp/package.json:14]: Should we start with x-beta.0?

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
       "**/react",
       "**/@react-navigation/**",
       "**/react/**",
+      "**/lodash**",
```
Comment (emdobrin) [inline package.json:15]: What was this change required for?

File: apps/AEPSampleApp/package.json (hunk 1 above)
Comment (emdobrin) [reply apps/AEPSampleApp/package.json:14]: For Assurance we also need to update the Readme: https://github.com/adobe/aepsdk-react-native/tree/staging/packages/assurance#install-npm-package

Comment (yangyansong-adbe) [reply apps/AEPSampleApp/package.json:14]: https://www.npmjs.com/package/@adobe/react-native-acpcore
https://www.npmjs.com/package/@adobe/react-native-aepcore
We used to release the alpha/beta version start from `.1`. Let me know if there is any reason we need to release a `beta.0` instead of a `beta.1`.

Comment (yangyansong-adbe) [reply apps/AEPSampleApp/package.json:14]: For Assurance, it's still a private package (not being released with other extensions). We can do more cleanup for Assurance in a separate PR after @PravinPK has updated the Gitbook.

File: package.json (hunk 2 above)
Comment (yangyansong-adbe) [reply package.json:15]: `lodash.isequal` is a new dependency of the `react-native-reanimated` package, we need to keep it in the sample app directory to make Metro bundle it at runtime.

**Review by cacheung — APPROVED** [review]

File: apps/AEPSampleApp/package.json (hunk 1 above)
Comment (emdobrin) [reply apps/AEPSampleApp/package.json:14]: 👍  
saw some occurrences with alpha.0 so I thought we can follow the same, but beta.1 sounds good then
